templates/documentation_layout.html

```diff
@@ -8,6 +8,8 @@
     <!-- The docs_config.js file is provided by HexDocs and shared
          between multiple versions of the same package. -->
     <script src="{{ unnest }}/docs_config.js"></script>
+    <link rel="stylesheet" href="//unpkg.com/@highlightjs/cdn-assets@10.5.0/styles/default.min.css">
+    <script src="//unpkg.com/@highlightjs/cdn-assets@10.5.0/highlight.min.js"></script>
```

Comment:
Oh this is cool!

I had a play around with this URL and I think we can pull the Gleam plugin from the CDN too

https://unpkg.com/highlightjs-gleam@0.2.2/dist/gleam.min.js

Reply:
I'll try it now